Team9Project/src/com/indragie/cmput301as1/ExpenseClaimAddActivity.java

```diff
@@ -129,6 +129,10 @@ private Intent constructResultIntent() {
 			user,
 			ExpenseClaim.Status.IN_PROGRESS
 		);
+		
```

Comment:
This code should be removed after you test it, because the actual implementation (it's in my `session` branch) uses a central observer that calls through to the API client rather than putting this everywhere we make changes.